src/views/DeleteAccount/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { Button, Stack, Typography } from '@mui/material';
+import { Box } from '@mui/system';
+import CancelButton from 'components/CancelButton';
+import { Trans, useTranslation } from 'react-i18next';
+
+const DeleteAccount = () => {
+  const [translation] = useTranslation();
+
+  return (
+    <Box component="main" maxWidth={400} margin="0 auto" padding={2}>
```

Comment:
Please don't style components inline, the code will get messy quickly.

Reply:
I have done it this way because of two reasons:
1. [Material-ui](https://mui.com/components/box/#system-props) allows styles to be used as attributes of the `Box` element,
2. in this particular case, our view doesn't have its own styles file. In my opinion, creating a file just to put 3 properties in there is pointless, especially that the Material-ui allows us to use CSS properties as normal tag attributes.